web-common/src/features/dashboards/url-state/filters/expression.ne

```diff
@@ -0,0 +1,51 @@
+@preprocessor esmodule
+# match an expression used in url filters
+# adapted from https://dev.mysql.com/doc/refman/5.7/en/expressions.html
+@builtin "whitespace.ne"
+@builtin "number.ne"
+@builtin "string.ne"
```

Comment:
Where are these referenced? I don't see any explicit references, but I'm guessing maybe the primitive types at the bottom (e.g. `sqstring`, `int`, `decimal`) are coming from here?

Reply:
Yes, the docs arent here for these right now. They point to the source code of these files: https://github.com/kach/nearley/tree/master/builtin